mne/io/pick.py

```diff
@@ -568,3 +576,35 @@ def _picks_by_type(info, meg_combined=False, ref_meg=False):
              ref_meg=ref_meg))
         )
     return picks_list
+
+
+def _check_excludes_includes(chs, info=None, allow_bads=False):
+    """Ensure that inputs to exclude/include are list-like or "bads".
+
+    Parameters
+    ----------
+    chs : any input, should be list, tuple, string
+        The channels passed to include or exclude.
+    allow_strs : list of strings
+        Optional strings to allow.
+
+    Returns
+    -------
+    chs : list
+        Channels to be excluded. If allow_bads, this will be the
+        bad channels found in 'info'.
```

Comment:
It's fine to pass includes...updated the docstring to be better.

On Tue, Aug 4, 2015 at 1:01 PM, Alexandre Gramfort <notifications@github.com
> wrote:

> In mne/io/pick.py
> <https://github.com/mne-tools/mne-python/pull/2375#discussion_r36234244>:
>
> > +
> > +def _check_excludes_includes(chs, info=None, allow_bads=False):
> > +    """Ensure that inputs to exclude/include are list-like or "bads".
> > +
> > +    Parameters
> > +    ----------
> > +    chs : any input, should be list, tuple, string
> > +        The channels passed to include or exclude.
> > +    allow_strs : list of strings
> > +        Optional strings to allow.
> > +
> > +    Returns
> > +    -------
> > +    chs : list
> > +        Channels to be excluded. If allow_bads, this will be the
> > +        bad channels found in 'info'.
>
> else it will be what? what if you pass include to _check_excludes_includes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2375/files#r36234244>.
>